.clang-format

```diff
@@ -38,7 +38,7 @@ BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: false
 BreakAfterJavaFieldAnnotations: false
 BreakStringLiterals: true
-ColumnLimit:     80
+ColumnLimit:     120
```

Comment:
It's because we have 120 for our rust code so we expect, that user doesn't use different resolutions for different languages :)

Reply:
Alright, I won't be stubborn here.